### PR TITLE
Release notes for Keycloak 20 - OpenBanking UK and FIPS

### LINKED ISSUE #1683: Removed Open Banking UK from the release notes for Keycloak 20
### Description

Release notes for the Keycloak 20 tasks, which I've merged:
- https://github.com/keycloak/keycloak/issues/12883 UK OpenBanking support

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

File: release_notes/topics/20_0_0.adoc [hunk 1]
```diff
@@ -40,3 +40,18 @@ Several deprecated methods were removed from data providers and models.
...
 link:{upgradingguide_link}[{upgradingguide_name}] for more details.
+
+= Open Banking UK support
+
+{project_name} is compliant with Open Banking UK. Details in the link:{adapterguide_link}#_open-banking-uk[{adapterguide_name}].
+Thanks to https://github.com/tnorimat[Takashi Norimatsu], who did the work on this.
+
+= FIPS 140-2 experimental support
+
+There is an effort in progress to support {project_name} on the FIPS 140-2 enabled environment. This is already possible with latest {project_name}. There will be a blog post with the details
```
Comment (stianst) [inline release_notes/topics/20_0_0.adoc]: You should highlight that it is experimental right now, as this could be interpreted as it's ready to go

Comment (mposolda) [reply release_notes/topics/20_0_0.adoc]: @stianst Good point. I've added "experimental" to the header, but not to the text itself. I've highlighted it a bit more in the text now. Can you please re-verify?

**Comment by stianst** [comment]
I've included the changes to the release notes in the following PR:
https://github.com/keycloak/keycloak-documentation/pull/1712

File: securing_apps/topics/oidc/fapi-support.adoc [hunk 2]
```diff
@@ -34,6 +34,24 @@ in the more strict way to enforce some of the requirements. Especially:
...
 
+[[_open-banking-uk]]
+==== Open Banking UK
+
+{project_name} is compliant with the https://www.openbanking.org.uk/[Open Banking UK]. On top of the <<_fapi-support,FAPI 1 Advanced>>, there is also requirement to
+use the `intent` concept to issue tokens.
+
+It works in a way that a client sends an authorization request with this intent. For example, UK OpenBanking requires the following steps:
+
+1. A client requests an access token in Client Credentials Grant, and an authorization server issues an access token.
```
Comment (stianst) [inline securing_apps/topics/oidc/fapi-support.adoc:45]:
```suggestion
1. A client retrieves an access token from Keycloak using Client Credentials Grant
```

**Review by stianst — APPROVED** [review]

**Pull request merged**